packages/react-focus-management/.npmignore

```diff
@@ -0,0 +1,34 @@
+*.api.json
+*.config.js
+*.log
+*.nuspec
+*.test.*
+*.yml
+.editorconfig
+.eslintrc*
+.eslintcache
+.gitattributes
+.gitignore
+.vscode
+coverage
+dist/storybook
+dist/*.stats.html
+dist/*.stats.json
+dist/demo
+fabric-test*
+gulpfile.js
+images
+index.html
+jsconfig.json
+node_modules
+results
+src/**/*
+!src/**/examples/*.tsx
+!src/**/docs/**/*.md
+!src/**/*.types.ts
```

Comment:
Not for this PR, but we should remove this block from the template in create-package (it's probably not relevant to the current setup)